core/domain/wipeout_service.py

```diff
@@ -483,6 +525,12 @@ def verify_user_deleted(
     if not auth_services.verify_external_auth_associations_are_deleted(user_id):
         return False
 
+    # Verify if user is deleted.
+    user_settings_model = user_models.UserSettingsModel.get_by_id(user_id)
+    if user_settings_model:
+        username = user_settings_model.username
+        _verify_profile_picture_is_deleted(username)
```

Comment:
This can only be done when `include_delete_at_end_models` is `False`, because when it is `True` we assume that ALL models were deleted so `UserSettingsModel` will not be accessible.

Reply:
Added this part with the condition when include_delete_at_end_models is False